servicecontrol/creating-config-file.md

```diff
@@ -257,6 +257,9 @@ Type: string
 
 Default: `<ErrorQueue>.log`
 
+Version 1.28 and below created the queue specified by this setting when the service instance is installed regardless of the value of `ServiceControl/ForwardErrorMessages`.  From V1.29 the queue is only created if `ServiceControl/ForwardErrorMessages` is enabled. 
```

Comment:
@gbiellem i know it is 50/50. both "Version 1.28 and below" and "Versions 1.28 and below" are correct and unambiguous. but when we threw it out for a vote most people preferred "Versions 1.28 and below". so that is what we push for in the interest of consistency